packages/ui/src/proposals/constants/styles.ts

```diff
@@ -2,7 +2,8 @@ import styled from 'styled-components'
 
 import { ListHeaders, ListHeader } from '@/common/components/List/ListHeader'
 
-export const ProposalColLayout = '296px 148px 156px 60px 1fr'
+export const ProposalColLayout = '296px 148px 156px 60px'
+export const ProposalColLayoutWithVoteDetails = ProposalColLayout + ' 80px'
```

Comment:
Can't the `60px` be removed ? Because this grid always seem to have an unused column.

For normal members:
![image](https://github.com/Joystream/pioneer/assets/6571453/8d7a0456-7d78-43e4-ad70-f55ae8570f65)

For CMs:
![image](https://github.com/Joystream/pioneer/assets/6571453/940de41d-e43f-4890-9679-f97217e84498)